doc/mgr/orchestrator.rst

```diff
@@ -180,9 +183,25 @@ Example::
     all-available-devices node2 /dev/vdc -  -
     all-available-devices node3 /dev/vdd -  -
 
-
 .. note::
-    Output form Cephadm orchestrator
+    Example output from Cephadm orchestrator
+
+When the parameter ``all-available-devices`` is used, a cephadm service is created.
+This service guarantees that all available devices in the cluster nodes will be used for OSD's.
+Take into account the implications of this behavior, which is automatic and enabled by default.
```

Comment:
again, this isn't bound to `all-available-devices`. It's the default behavior of `cephadm` in general, which can only be changed by adding the `unmanaged: True` flag